### ISSUE TITLE
No Seq Logging seen

### ISSUE THREAD

**Issue by raybooysen** [datalust/bunyan-seq#4]
Hi there

I've configured bunyan with three stream.  A console stream, a file stream and a bunyan-seq stream.  I'm seeing the console logs and the file stream log, but nothing is being logged to seq.

Nothing is being logged to the console for the onError handler, and no traffic is leaving the client (verified with Wireshark).  What else can I do to debug this issue?

sample config:

```
seq.createStream({
            serverUrl: 'http://seq:5432',
            apiKey: 'APIKEY,
            level: 'debug' 
});
```

**Comment by nblumhardt** [comment]
Hi Ray, thanks for dropping by! Guessing that port number is just for example? (The default's `5341`.)

Is it a long-running (e.g. server) app? Which Node version are you running?

**Comment by nblumhardt** [comment]
Hi Ray, I've just pulled in your fix continuousit/seq-logging#1 and republished `bunyan-seq@1.2` to NPM. Fingers crossed that was the issue here, let me know if it's not resolved. Cheers!

**Comment by raybooysen** [comment]
We fixed the issue which was down to our own issues on the server side.  We had the seq POSTs being rejected.  However, the onError was never being called and nothing logged to the console (which is the default of onError).  Once I have a test case I'll open an issue which proves this specific case.
